xml/http_servers.xml

```diff
@@ -1,6 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <fingerprints matches="http_header.server" protocol="http" database_type="service" preference="0.90">
   <!-- HTTP Server headers are matched against these patterns to fingerprint HTTP servers. -->
+  <fingerprint pattern="(?i)^AirTunes/([\d\.]+)$">
+    <description>Apple AirTunes/AirPlay, more generally RTSP used by a variety of wireless a/v products</description>
+    <example service.version="220.68">AirTunes/220.68</example>
+    <param pos="0" name="service.vendor" value="Apple"/>
+    <param pos="0" name="service.product" value="AirTunes"/>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="hw.device" value="Media Server"/>
```

Comment:
At some point we should look at if these are source or sinks as well as if this banner can be seen on Macs or Windows boxes running iTunes, Kodi, etc.

Reply:
My understanding is that this banner will be shown by any device speaking the protocol used by AirTunes, which is largely iTunes/other apple stuff but will definitely also include other vendors/products.  That is why I kept it generic; we know the protocol and can suggest something about the device.